.github/workflows/integration-test.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
           provider: microk8s
           channel: 1.29-strict/stable
           juju-channel: 3.4/stable
-          lxd-channel: 5.20/stable
+          lxd-channel: 5.21/stable
```

Comment:
Might be a good idea to bump to microk8s 1.31-strict/stable and juju 3.6 at the same time.